backend/internal/data/migrations/sqlite3/20250706190000_generate_migration.py

```
@@ -0,0 +1,59 @@
+  WHERE type = 'table'
+    AND name NOT LIKE 'sqlite_%'
+)
+
+SELECT
+  '["' || t.table_name || '", "' || c.name || '"],' AS table_column
+FROM tables t
+JOIN pragma_table_info(t.table_name) c
+WHERE c.name like'%date%'; """
+
+fields = [["auth_tokens", "created_at"],
+["auth_tokens", "updated_at"],
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)